opensrp-app/src/main/java/org/smartregister/ssl/OpensrpSSLHelper.java

```diff
@@ -70,15 +76,15 @@ public void verify(String host, String[] cns, String[] subjectAlts) throws
             KeyStore trustedKeystore = KeyStore.getInstance(KeyStore.getDefaultType());
             InputStream inputStream = context.getResources().openRawResource(R.raw.opensrp_truststore);
             try {
-                trustedKeystore.load(inputStream, "phone red pen".toCharArray());
+                trustedKeystore.load(inputStream, PASS.toCharArray());
             } finally {
                 inputStream.close();
             }
             //Initialise a TrustManagerFactory with the CA keyStore
-            TrustManagerFactory tmf = TrustManagerFactory.getInstance("X509");
+            TrustManagerFactory tmf = TrustManagerFactory.getInstance(X509);
```

Comment:
Better to use `TRUST_MANAGER_ALGORITHM`  in case `X509` is not preferred in future?

Reply:
This has been moved to build config